app/assets/javascripts/discourse/app/models/topic.js

```diff
@@ -284,25 +277,6 @@ const Topic = RestModel.extend({
     return userPath(username);
   },
 
-  // The amount of new posts to display. It might be different than what the server
-  // tells us if we are still asynchronously flushing our "recently read" data.
```

Comment:
This doesn't make sense to me in the current time and age of reading the code since the session prop `highestSeenByTopic` is set asynchronously while flushing our recently read data.